constants/requests.ts

```
@@ -26,6 +26,9 @@ export const REQUEST_LOG_TYPE = {
   REQUEST_CANCELLED: "REQUEST_CANCELLED",
   REQUEST_UPDATED: "REQUEST_UPDATED",
   PENDING_REQUEST_FOUND: "PENDING_REQUEST_FOUND",
+  REQUEST_ALREADY_APPROVED: "REQUEST_ALREADY_APPROVED",
+  REQUEST_ALREADY_REJECTED: "REQUEST_ALREADY_REJECTED",
+  //comment for testing purposes
```

ESLint

[error] Expected space or tab after '//' in comment.

(spaced-comment)